apps/src/assetManagement/IconList.jsx

```diff
@@ -0,0 +1,84 @@
+var IconListEntry = require('./IconListEntry.jsx');
+var icons = require('./icons').aliases;
```

Comment:
I wonder if we should call this variable something like iconAliases or maybe iconTree? I was expecting this to be a list until I stepped through in the debugger.